static/usage/popover/presenting/inline-trigger/vue.md

```diff
@@ -0,0 +1,29 @@
+```html
+<template>
+  <div>
```

Comment:
Do we need this div here?

Reply:
Yes; if you remove it in Stackblitz, you can see the buttons stretch across the screen oddly.

Reply:
Might be better to use an `ion-content` instead of `div` here.